skimage/restoration/_denoise.py

```diff
@@ -54,6 +54,18 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
     ----------
     .. [1] http://users.soe.ucsc.edu/~manduchi/Papers/ICCV98.pdf
 
+    Example
+    -------
+    >>> from skimage import data, img_as_float
+    >>> from skimage.restoration import denoise_bilateral
+    >>> import numpy as np
+    >>>
+    >>> astro = img_as_float(data.astronaut())
+    >>> astro = astro[220:300, 220:320]
+    >>>
+    >>> noisy = astro + 0.6 * astro.std() * np.random.random(astro.shape)
+    >>> noisy = np.clip(noisy, 0, 1)
+    >>> denoised = denoise_bilateral(noisy, sigma_range=0.05, sigma_spatial=15)
```

Comment:
I think that ``sigma_spatial=15`` is a very large value, since the ``window_size`` is 5 by default. Maybe put a smaller value? Also, adding a second line with a larger value of ``sigma_range`` might be interesting, with a comment about the effect of ``sigma_range``.